scripts/helpers.js

```diff
@@ -109,12 +113,21 @@ hexo.extend.helper.register('github_file_url', function (path) {
   return urljoin(this.config.github.aframe.url, 'blob', aframeCurrentSha, path);
 });
 
+/**
+ * Get GitHub edit URL.
+ * - Change .html to .md.
+ * - For docs, must remove the active version (e.g., docs/0.2.0/guide.html -> docs/guide.md).
+ */
 hexo.extend.helper.register('website_github_edit_url', function (path) {
+  // For docs.
   if (path.indexOf('docs/') !== -1) {
-    return urljoin(this.config.github.aframe.url, 'edit', 'master', path.replace(/\.html$/, '.md'));
-  } else {
-    return urljoin(this.config.github.aframe_site.url, 'edit', 'master', this.config.source_dir, path.replace(/\.html$/, '.md'));
+    var activeVersion = path.split('/')[path.split('/').indexOf('docs') + 1];
```

Comment:
ah, you can use some combination of …

* [Hexo Page Variables](https://hexo.io/docs/variables.html#Page-Variables)
* [Hexo Data](https://github.com/aframevr/aframe-site/blob/75725da/scripts/generators.js#L80)
* [Front Matter](https://github.com/lalunamel/hexo-front-matter-excerpt/blob/master/index.js) (for the metadata you care about) injected for the Docs pages

maybe all the helper functions that extract info from the URL are fine. if you think it's manageable to keep things as is, by all means.